src/FamilyHubs.ServiceDirectory.Infrastructure/Services/PostcodesIo/PostcodesIoResponse.cs

```diff
@@ -3,7 +3,11 @@
 
 namespace FamilyHubs.ServiceDirectory.Infrastructure.Services.PostcodesIo;
 
-//todo: can we flatten the result during deserialization?
-public sealed record PostcodesIoResponse(
-    [property: JsonPropertyName("error")] string? Error,
-    [property: JsonPropertyName("result")] PostcodeInfo PostcodeInfo);
+public sealed class PostcodesIoResponse
+{
+    [JsonPropertyName("error")]
+    public string? Error { get; set; }
+
+    [JsonPropertyName("result")]
+    public PostcodeInfo Result { get; set; } = default!;
```

Comment:
`PostcodeInfo` as a name is a bit more descriptive.

Reply:
there is no name change for `PostcodeInfo` class in this pr